File: library/src/main/java/cn/wandersnail/common/http/download/MultiDownloadListener.java
package cn.wandersnail.common.http.download;

import cn.wandersnail.common.http.callback.MultiProgressListener;

/**
 * 多任务下载监听器
 * <p>
 * date: 2019/8/23 15:40
 * author: zengfansheng
 */
public interface MultiDownloadListener<T extends DownloadInfo> extends DownloadListener<T>, MultiProgressListener {
}

File: library/src/main/java/com/snail/java/network/download/MultiDownloadListener.kt
package com.snail.java.network.download

import com.snail.java.network.callback.MultiProgressListener

/**
 * 多下载任务监听
 *
 * date: 2019/2/28 20:26
 * author: zengfansheng
 */
interface MultiDownloadListener<T : DownloadInfo> : DownloadListener<T>, MultiProgressListener